source/indexer/test/Indexer-unit.js

```diff
@@ -112,7 +116,11 @@ contract('Indexer Unit Tests', async accounts => {
       })
 
       notEmitted(result, 'CreateIndex', event => {
-        return event.signerToken === tokenOne && event.senderToken === tokenTwo
+        return (
+          event.signerToken === tokenOne &&
+          event.senderToken === tokenTwo &&
+          event.createdIndex !== EMPTY_ADDRESS
```

Comment:
I reckon instead check the actual address.
i.e.
```
const result = await indexer.createIndex(tokenOne, tokenTwo, {
      from: aliceAddress,
})

const index = await indexer.indexes(tokenOne, tokenTwo)
```
Then you can update
`event.createdIndex !== EMPTY_ADDRESS` to `event.createdIndex === index.address`